src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -134,7 +139,7 @@ private void OnFileSaved(object sender, TextDocumentFileActionEventArgs e)
 
                         Logger.LogEvent(textMessage, LogLevel.Task);
                         Logger.LogEvent(ex.ToString(), LogLevel.Error);
-                        Telemetry.TrackException("restoresavefailed", ex);
+                        Telemetry.TrackException("RestoreOnSaveFailed", ex);
```

Comment:
>RestoreOnSaveFailed [](start = 50, length = 19)

Fail-RestoreOnSave

Reply:
In many cases we are using the method name which is easier to identify the source

---
In reply to: [200786175](https://github.com/aspnet/LibraryManager/pull/282#discussion_r200786175) [](ancestors = 200786175)